megamek/src/megamek/server/Server.java

```diff
@@ -3135,71 +3135,78 @@ private void receiveForwardIni(int connectionId) {
         if (game.getOptions().booleanOption(OptionsConstants.RPG_INDIVIDUAL_INITIATIVE)) {
             return;
         }
+        
         // get the next player from the team this player is on.
         IPlayer next = game.getTeamForPlayer(current).getNextValidPlayer(current, game);
```

Comment:
```suggestion
        // get the team for the next player
        Team team = game.getTeamForPlayer(current)
        // Create the player
        IPlayer next;
        //the team can theoretically be null, so we need to check and ensure it is not a null value
        if (team != null) {
            // get the next player from the team this player is on.
            next = game.getTeamForPlayer(current).getNextValidPlayer(current, game);
        } else {
            next = null;
        }
```

Reply:
This will properly handle the potential case of a null team, which shouldn't happen but the code for getTeamForPlayer says is a possibility.